pycbc/workflow/plotting.py

```diff
@@ -128,15 +128,25 @@ def make_throughput_plot(workflow, insp_files, out_dir, tags=None):
     workflow += node
 
 def make_foreground_table(workflow, trig_file, bank_file, ftag, out_dir,
-                          singles=None, extension='.html', tags=None):
-    if tags is None:
+                          singles=None, extension='.html', tags=None,
+                          hierarchical_level=None):
+
+    if hierarchical_level and tags:
+        tags = [("HIERARCHICAL_LEVEL_{num:02d}".format(
+                hierarchical_level))] + tags
+    elif hierarchical_level and not tags:
+        tags = ["HIERARCHICAL_LEVEL_{num:02d}".format(hierarchical_level)]
+    elif not hierarchical_level and not tags:
         tags = []
+
     makedir(out_dir)
     node = PlotExecutable(workflow.cp, 'page_foreground', ifos=workflow.ifos,
                     out_dir=out_dir, tags=tags).create_node()
     node.add_input_opt('--bank-file', bank_file)
     node.add_opt('--foreground-tag', ftag)
     node.add_input_opt('--trigger-file', trig_file)
+    if hierarchical_level: node.add_opt('--plot-hierarchical-level',
+                                        hierarchical_level)
```

Comment:
Shouldn't this be `--use-hierarchical-level` or vice-versa?

Reply:
fixed, thanks